data/card_catalogue/greys.json

```diff
@@ -14,8 +17,11 @@
     "upgrade": "",
     "tier": "grey",
     "details": {
-      "rules": "Move 1 square in a straight line",
-      "flavour": "When speed gets in the blood, one must move to live",
+      "rules": {
+        "active": "Move a metre.",
+        "passive": ""
+      },
+      "flavour": "When speed gets in the blood, one must move to live.",
```

Comment:
I think this carries every sort of implication we need to to have, especially along with the image:

<p align="center">
<img src="https://user-images.githubusercontent.com/8680290/127941236-751beb93-ac1f-4605-91eb-8e6364d08cfb.png"  width="50%" title="Shift (Alternative Format)"></p>

Reply:
I like this wording, though it does get a little weird on the diagonal. I think it makes sense to leave the diagonal case up to DM interpretation. Especially if someone tries to play the game with real distances rather than using a grid, then they might want to interpret this card entirely literally since they can handle the diagonal case

Reply:
❗  🗨️ *"Get back here with my 414 centimetres!"*
🏃  🚕 💨